src/viam/sdk/robot/client.cpp

```diff
@@ -354,7 +354,7 @@ std::shared_ptr<RobotClient> RobotClient::at_local_socket(const std::string& add
     const std::string addr = "unix://" + address;
     const char* uri = addr.c_str();
     const std::shared_ptr<grpc::Channel> channel =
-        grpc::CreateChannel(uri, grpc::InsecureChannelCredentials());
+        viam::sdk::impl::create_viam_channel(uri, grpc::InsecureChannelCredentials());
```

Comment:
You probably don't need the full qualification here.

Reply:
you're right, I think I can get away with `sdk::impl` but `impl` alone was erroring out because `RobotClient` uses the pimpl idiom with `struct impl`